App/src/Service/GroupExpenseBalancer.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Bilan;
+use App\Entity\Expense;
+
+class GroupExpenseBalancer
+{
+    public function expenseBalancer($expenses)
+    {
+        /**
+         * @var Bilan[]
```

Comment:
Here you indicate that you have a data structure like `[Bilan, Bilan, ...]` however you define something like `["name" => Bilan, "name" => Bilan, ...]`. You can read this documentation from [phpstan](https://phpstan.org/writing-php-code/phpdoc-types) which actually works for most of IDEs 👍

Reply:
Okay :)